src/vmm/src/devices/virtio/queue.rs

```diff
@@ -618,9 +618,7 @@ impl Queue {
     /// successfully enabled. Otherwise it means that one or more descriptors can still be consumed
     /// from the available ring and we can't guarantee that there will be a notification. In this
     /// case the caller might want to consume the mentioned descriptors and call this method again.
-    pub fn try_enable_notification<M: GuestMemory>(&mut self, mem: &M) -> bool {
-        debug_assert!(self.is_valid(mem));
-
```

Comment:
Please, merge this commit with the change that makes with the commit that makes `mem` unused

Reply:
I would prefer to keep these separate. Code without this commit is technically valid. And squashing it with `refactor(queue): remove set_used_ring_avail_event method` will only make it more complicated.